node/src/actors/chain_manager/mod.rs

```diff
@@ -2174,14 +2174,32 @@ fn show_info_dr(data_request_pool: &DataRequestPool, block: &Block) {
         .data_request_pool
         .iter()
         .fold(String::new(), |acc, (k, v)| {
-            format!(
-                "{}\n\t* {} Stage: {}, Commits: {}, Reveals: {}",
-                acc,
-                White.bold().paint(k.to_string()),
-                White.bold().paint(format!("{:?}", v.stage)),
-                v.info.commits.len(),
-                v.info.reveals.len()
-            )
+            if v.stage == DataRequestStage::COMMIT || v.stage == DataRequestStage::REVEAL {
+                let current_round = if v.stage == DataRequestStage::COMMIT {
+                    v.info.current_commit_round
+                } else {
+                    v.info.current_reveal_round
+                };
+                format!(
+                    "{}\n\t* {} Stage: {} ({}/{}), Commits: {}, Reveals: {}",
+                    acc,
+                    White.bold().paint(k.to_string()),
+                    White.bold().paint(format!("{:?}", v.stage)),
+                    current_round,
+                    data_request_pool.extra_rounds,
```

Comment:
Since this is "extra" rounds, the total number of rounds is 1 more:

```suggestion
                    data_request_pool.extra_rounds + 1,
```

Without this change it looks like

```
    	* 32bccb369c43191532e70bf95dc48b4e61f657c6b82c12bd4f0a06976f3d819e Stage: COMMIT (1/3), Commits: 0, Reveals: 0
    	* 32bccb369c43191532e70bf95dc48b4e61f657c6b82c12bd4f0a06976f3d819e Stage: COMMIT (2/3), Commits: 0, Reveals: 0
    	* 32bccb369c43191532e70bf95dc48b4e61f657c6b82c12bd4f0a06976f3d819e Stage: COMMIT (3/3), Commits: 0, Reveals: 0
    	* 32bccb369c43191532e70bf95dc48b4e61f657c6b82c12bd4f0a06976f3d819e Stage: COMMIT (4/3), Commits: 0, Reveals: 0
    	* 32bccb369c43191532e70bf95dc48b4e61f657c6b82c12bd4f0a06976f3d819e Stage: TALLY, Commits: 0, Reveals: 0
```

Reply:
Right, did not take into account that it runs up to the extra rounds (and is not strictly smaller than that parameter).